src/Example/ProjectParser.cs

```diff
@@ -46,8 +46,7 @@ public ProjectInformation Parse(FilePath path)
             // Got an order?
             var order = 1024;
             var orderString = Parse(xml, "//ExampleOrder");
-            if (string.IsNullOrWhiteSpace(orderString) &&
-                int.TryParse(orderString, out var orderInteger))
+            if (int.TryParse(orderString, out var orderInteger))
```

Comment:
Great catch!

We want to make sure to only do the comparison if `orderString` is not null or whitespace though, otherwise the sort order will be `0` instead of `1024` by default.

Reply:
How do you see it being `0` when `order` will never be assigned `orderInteger` if `int.TryParse` returns `false`? I see what you're saying to be the case if `order` was the output variable, as in `int.TryParse(orderString, out var order)`, but it's a new variable.

Reply:
Ah, correct, didn't read properly.